hack/local-up-cluster.sh

```diff
@@ -570,6 +570,7 @@ EOF
       --service-account-key-file="${SERVICE_ACCOUNT_KEY}" \
       --service-account-lookup="${SERVICE_ACCOUNT_LOOKUP}" \
       --service-account-issuer="https://kubernetes.default.svc" \
+      --service-account-jwks-uri="https://kubernetes.default.svc/openid/v1/jwks" \
```

Comment:
for the curious, this is because the JWKS URL published in the OIDC discovery doc defaults to using the `--external-hostname`, expecting that to be generally resolveable.

local-up-cluster sets `--external-hostname=localhost` which is... not generally resolveable.

Reply:
neat